analysis/SConstruct

```diff
@@ -48,7 +48,9 @@ Default('./build', './release')
 if mode == 'cache':
     CacheDir(cache_dir)
 
-debrief_env = {'MAXIT' : 10,
+# Debrief args is a dictionary storing the arguments that are useful for functions to run 
```

Comment:
I still find the dictionary is not very easy to read, since it contains too many lines with comments in it. Can we have one line for one argument, add comments right after the argument (not in a separate line), and make the `:` vertically aligned?